docs/src/design/Patterns.js

```diff
@@ -101,7 +103,7 @@ var Patterns = React.createClass({
     </div>
 
   </section>
-      </article>
+      </Article>
```

Comment:
I know it used to be like this before. But would you mind to fix the spacing for this file?